open_humans/models.py

```diff
@@ -26,21 +26,21 @@ def get_member_profile_image_upload_path(instance, filename):
     """
     Construct the upload path for a given member and filename.
     """
-    return 'member/%s/profile-images/%s' % (instance.user.id, filename)
+    return str('member/{0}/profile-images/{1}').format(instance.user.id, filename)
```

Comment:
I noticed that we're giving the `{0}/{1}.format(foo,bar)` formatting in some places and in others only the `{}/{}` notation. Is there a reason why?

Reply:
No real reason.  I'll make them consistent.

Reply:
nice, thanks!